### PR TITLE
Add CI to run postman tests

### PR DESCRIPTION
**Pull request by brownoxford**
Update `ci.yml` action to include postman tests. Note that this PR is contingent on #73 being merged first.

FIX: #68

### CODE REVIEW

**Comment by brownoxford** [comment]
Note that the `if` condition for the new `postman` tests isn't quite working right as it ran for this PR. I'll create a separate issue for that.

**Review by mprorock — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,4 +15,26 @@ jobs:
...
+          node-version: 16.x
+      - name: Install dependencies
+        run: npm ci
+        working-directory: packages/tests-postman
+      - name: Run Postman tests
+        run: node .
+        working-directory: packages/tests-postman
+        env:
+          CLIENT_SECRET_VC_MESUR_IO: ${{ secrets.CLIENT_SECRET_VC_MESUR_IO }}
+          CLIENT_SECRET_b0643936e30843b622c78ad353da19123e4583817491fe05e2e9d39824983bee: ${{ secrets.CLIENT_SECRET_b0643936e30843b622c78ad353da19123e4583817491fe05e2e9d39824983bee }}
```
Comment (mkhraisha) [inline .github/workflows/ci.yml:40]: similar to my comment on the other PR, would much prefer this to be `CLIENT_SECRET_TRANSMUTE`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -15,4 +15,26 @@ jobs:
...
+        uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - name: Install dependencies
+        run: npm ci
+        working-directory: packages/tests-postman
+      - name: Run Postman tests
+        run: node .
```
Comment (mkhraisha) [inline .github/workflows/ci.yml:36]: nit: this should be `npm run test:all`

also any specific reason we're cding into the packages file and not just running it from base with lerna?

Comment (brownoxford) [reply .github/workflows/ci.yml:36]: Nope, just my lack of knowledge on using lerna :) Looking into fixing this now.

Comment (brownoxford) [reply .github/workflows/ci.yml]: ok, updated scripts to use lerna.

**Review by OR13 — APPROVED** [review]

**Pull request merged**